app/components/meeting-schedule/component.js

```diff
@@ -47,6 +47,7 @@ export default Ember.Component.extend({
                     retList.push([i]);
                 }
             });
+            this.set('selectedItemId', retList[0][0].id);
```

Comment:
Then you shouldn't be using the schedule component ;) just kidding, I'll add a check for that!